tests/www/search/tests.py

```diff
@@ -566,15 +569,36 @@ def test_contract_type(self):
             company=company, appellation=appellations[1], contract_type=ContractType.BUSINESS_CREATION
         )
 
-        inactive_company = CompanyFactory(
-            department="45", coords=city.coords, post_code="44117", kind=CompanyKind.EI, convention=None
-        )
+        # Start active to verify it appears in search results.
+        inactive_company = CompanyFactory(department="45", coords=city.coords, post_code="44117", kind=CompanyKind.EI)
```

Comment:
```suggestion
        other_company = CompanyFactory(department="45", coords=city.coords, post_code="44117", subject_to_eligibility=True)
```
Ça sera peux-être plus résistant au futur relecture et au "Mais `EI` c'est la valeur par défaut, sont idiots !" :grin:.

Et le "inactive" alors qu'on veux justement quelle soit active m'a trigger, déso :hugs:. (y a le même dans `test_domains`)

Reply:
Je ne touche pas au kind dans cette PR. Un chantier après l’autre ;)